addons/mail/models/mail_blacklist.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class MailBlackList(models.Model):
+    """ Model of blacklisted email addresses to stop sending emails."""
+    _name = 'mail.blacklist'
+    _description = 'Mail Blacklist'
+    _order = 'email'
+    _rec_name = 'email'
+
+    email = fields.Char(string='Email Address', required=True)
```

Comment:
Hello Yajo,

Thanks for your kind suggestion. As If you could take a look in  "_sql_constraints" constraint part. It will going to create unique constraint in database. So According to db behaviour it will automatically create index on given column. So we do not require to specify individual index attribute to create index.(Like primary key behave in DB model)

Thanks & Regards